taxi/views.py

```diff
@@ -1,3 +1,16 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
 
-# Create your views here.
+from .models import Car, Driver, Manufacturer
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_cars = Car.objects.all().count()
+    num_drivers = Driver.objects.all().count()
+    num_manufacturers = Manufacturer.objects.all().count()
+    context = {
```

Comment:
Since you use those variables only once you could call it directly in the context without defining unnecessary variables